src/OpenSage.Tools.BigEditor/UI/MainForm.cs

```diff
@@ -15,15 +16,21 @@ internal sealed class MainForm : DisposableBase
         private BigArchive _bigArchive;
         private readonly List<BigArchiveEntry> _files;
         private int _currentFile;
+        private string _currentFileName;
         private string _currentFileText;
 
         private byte[] _searchTextBuffer;
         private byte[] _filePathBuffer;
         private string _searchText;
+        static float _scrollY;
+
+        private static NLog.Logger logger = NLog.LogManager.GetCurrentClassLogger();
 
         public MainForm()
         {
             _files = new List<BigArchiveEntry>();
+            _currentFileName = "";
```

Comment:
Maybe this should be `null` when no file is selected?

Reply:
Maybe) Changed